src/Logger.ts

```diff
@@ -10,6 +10,11 @@ import { toJSON } from './utils/toJSON';
 
 const ERROR_EVENT = 'type.googleapis.com/google.devtools.clouderrorreporting.v1beta1.ReportedErrorEvent';
 
+/**
+ * A function type which can format a log entry to a string
+ */
+type LogFormatter = (entry: LogEntry) => string
```

Comment:
```suggestion
export type LogFormatter = (entry: LogEntry) => string;
```

Reply:
Let's re-export this type in `index.ts` to make it available for use. You can also create a file for this type alone.

Reply:
Done.